tools.go

```diff
@@ -1,13 +1,14 @@
-// +build tools
+//go:build tools
```

Comment:
`//go:build` was introduced in 1.17 but we're still bound to compile with 1.15 - isn't that going to disrupt the compilation process if 1.15 is used?

Reply:
Tested with 1.15, and it compiles fine.